.github/workflows/build.yml

```diff
@@ -0,0 +1,124 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Fozzie Components CI
+
+on:
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
+    steps:
+    # Checkout the Repo
+    - name: Checkout
```

Comment:
You'll notice some steps (`checkout` / `setup-node` etc) are duplicated across jobs.

This is because each job runs inside of its own container. Unlike CircleCI, GitHub Actions as far as I can tell, doesn't have the concept of persisting the state of the container for use in later jobs.

We do utilise caches though, so these steps shouldn't add any build time.

We also need to find a way to reuse these steps across jobs to reduce duplication.